lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -42,7 +43,9 @@ def precompile_assets
 
       def precompile_sti_loader
         Dir.chdir(miq_dir) do
-          shell_cmd("bundle exec rake evm:compile_sti_loader")
+          shell_cmd("BUNDLER_GROUPS=manageiq_default,ui_dependencies,graphql_api bundle exec rake evm:compile_sti_loader")
```

Comment:
Figured out the "random" graphql constants showing up.  Only two of the workers include the graphql_api bundler group (UI and web_service) so on the first run after a dnf upgrade every worker thinks it has to write the file out.  Only if one of these two workers exit last will the file end up with GraphQL constants.

After the first run every worker except UI/WS think they have a clean cache so only these two workers will write the file out.

Reply:
is there an `"all"` bundler group?

Reply:
I didn't want to go that route since the build machine is unlikely to have `ENV["APPLIANCE"]` which means we would pull in the `group :development, :test` stuff.